proto/cosmos/slashing/genesis.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+package cosmos.slashing;
+
+option go_package            = "github.com/cosmos/cosmos-sdk/x/slashing/types";
+
+import "gogoproto/gogo.proto";
+import "cosmos/slashing/slashing.proto";
+
+// GenesisState - all slashing state that must be provided at genesis
+message GenesisState {
+  Params params = 1 [
+    (gogoproto.casttype) = "Params",
+    (gogoproto.nullable) = false
+  ];
+
+  repeated SigningInfo signing_info = 2 [
+    (gogoproto.moretags) = "yaml:\"signing_info\"",
```

Comment:
Oops, maybe my comment in https://github.com/cosmos/cosmos-sdk/pull/6881#discussion_r463519001 wasn't very clear: the type should be singular, the field-name plural.

so:
- I would put `SigningInfo` (no s) & `signing_infos` (with s) here
- the `SigningInfo` and `ValidatorSigningInfo` below are correct